linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/H2Config.scala

```diff
@@ -198,22 +196,15 @@ trait H2SvcConfig extends SvcConfig {
     )
 
   @JsonProperty("responseClassifier")
-  var _h2ResponseClassifier: Option[H2StreamClassifierConfig] = None
+  var _h2ResponseClassifier: Option[H2ClassifierConfig] = None
 
   @JsonIgnore
-  def h2BaseResponseClassifier = H2StreamClassifiers.Default
+  def h2ResponseClassifier: Option[H2Classifier] =
```

Comment:
This field should probably be renamed to `h2Classifier` if the class is being renamed...